SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -444,6 +449,16 @@ TArray<UClass*> GetAllSupportedClasses()
 			continue;
 		}
 
+		// Avoid processing classes contained in Directories to Never Cook
+		const FString& ClassPath = SupportedClass->GetPathName();
+		if (DirectoriesToNeverCook.FindByPredicate([&ClassPath](const FDirectoryPath& Directory)
```

Comment:
Could you break this out calling the `FindByPredicate` and the `if` into two statements for readability?

Reply:
Changed to ContainsByPredicate which should be clear enough.